src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/scripted_fields_table/components/call_outs/call_outs.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import {
+  EuiCallOut,
+  EuiLink,
+  EuiSpacer,
+} from '@elastic/eui';
+
+export const CallOuts = ({
+  deprecatedLangsInUse,
+  painlessDocLink,
+}) => {
+  if (!deprecatedLangsInUse.length) {
+    return null;
+  }
+
+  return (
+    <div>
+      <EuiCallOut
+        title="Deprecation Warning"
```

Comment:
According to our [writing guidelines](https://elastic.github.io/eui/#/writing), this should be sentence case, "Deprecation warning." @gchaps Could you look over the copy below and provide feedback?

Reply:
Sure!